snow/consensus/snowman/topological.go

```diff
@@ -182,16 +181,6 @@ func (ts *Topological) Add(blk Block) error {
 	return nil
 }
 
-func (ts *Topological) Decided(blk Block) bool {
-	// If the block is decided, then it must have been previously issued.
-	if blk.Status().Decided() {
-		return true
-	}
-	// If the block is marked as fetched, we can check if it has been
-	// transitively rejected.
-	return blk.Status() == choices.Processing && blk.Height() <= ts.lastAcceptedHeight
```

Comment:
The `choices.Processing` check was previously needed because block's used to have a `Parent` method that returned a `snowman.Block` which could potentially have a status of `choices.Unknown` (if the parent wasn't known). But we no longer have this weird mechanism.